src/Directory.Build.props

```
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <Copyright>2015-2024 The Neo Project</Copyright>
-    <LangVersion>12.0</LangVersion>
+    <LangVersion>13.0</LangVersion>
```

```suggestion
    <LangVersion>latest</LangVersion>
```

Now we never have to update it.